components/autocomplete/SelectedItem.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../icon/Icon';
+
+const SelectedItem = ({ item, onRemove }) => {
+    const statusClasses = item.invalid ? 'autocomplete-selectedItem-invalid' : '';
+    return (
```

Comment:
What is this component about ?

Reply:
Show which values are picked from the list. I added a screenshot in the description.